packages/lit-dev-content/site/docs/ssr/overview.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Server-side rendering (SSR)
+eleventyNavigation:
+  key: 🧪 Overview
+  parent: Server-side rendering
+  order: 1
+---
+
+<details class="pre-release">
+  <summary> 🧪 Pre-release software: not for production.</summary>
+  
+  Lit labs packages are published to get feedback from the wider community. This code shouldn't be used in production, but we encourage you to try it out and [file issues](https://github.com/lit/lit/issues/new/choose) for any bugs you find. For general feedback, please use the GitHub [discussion](https://github.com/lit/lit/discussions).
+
+  For more information about the Lit labs process, see [Lib Labs](/docs/libraries/labs/)
+
+</details>
+
+Server-side rendering (SSR) can help make pages work without JavaScript, either for robustness or performance reasons, or to work with web crawlers and bots that don't run a full browser. Some sites can render faster if they render static markup first, and optionally hydrate the components afterwards by loading the JavaScript.
```

Comment:
We need to be specific with what SSR can do. It doesn't make pages "work" without JavaScript, but _render_ without them. From there the page can be made to work with hydration, or left to be used w/o JS.

Here's an attempt to use lists to make the paragraph more scannable:

```suggestion
Server-side rendering (SSR) can help make pages display without JavaScript by deeply rendering the HTML and CSS of components on the server.

SSR is used for a variety of reasons:
1. Robustness: to make pages render even if the JavaScript fails to load or the user has JavaScript disabled.
2. Performance: Some sites can render faster if they render static HTML first without wanting for JavaScript to load, then (optionally) load the page's JavaScript and hydrate the components.
3. SEO and web crawlers: While the major search-engine web crawlers render pages with full JavaScript-enabled browsers, not all web crawlers support JavaScript.
```

@arthurevans

Reply:
I like this. I'd suggest a couple of minor tweaks to Justin's suggestion.

"SSR is used" => "You can use SSR ..."

Numbered list => bulleted list. Order isn't really significant here, so an unordered list makes more sense.

Elsewhere on the site, we almost invariably format a list like this with a period rather than a colon after initial word/clause. That is:

-  Robustness. To make pages render even if the JavaScript fails to load or the user has JavaScript disabled.

To make that first one into a sentence, might change it to:

-  Robustness. Static HTML renders even if the JavaScript fails to load or the user has JavaScript disabled.

And in the second list item: "wanting" => "waiting"